crates/client/sync/src/eth/src/config.rs

```diff
@@ -0,0 +1,89 @@
+
```

Comment:
All of this config will need to move in the very near future to the `ChainConfig` struct I'm introducing in #198.
This is fine for now as #198 isnt merged yet but the change will need to be done if it gets merged before this one.